README.md

```diff
@@ -1 +1,88 @@
-# py-blast-dbf
+# pyreaddbc
+
+**pyreaddbc** is a Python library for working with DBC (Dengue Brazilian Case Registry) and DBF (dBase) files. It is an extension used in the PySUS package, which provides access to Brazilian public health data. For more information, you can visit the [PySUS repository](https://github.com/AlertaDengue/pysus).
+
+## Installation
+
+You can install **pyreaddbc** using pip:
+
+```bash
+pip install pyreaddbc
+```
+
+## Usage
+
+### Reading DBC Files
+
+To read a DBC file and convert it to a pandas DataFrame, use the `read_dbc` function:
```

Comment:
Add this sentence:  "Extracting the DBF from a DBC may require to specify the encoding of the original data, if known."

Reply:
Done!

Reply:
Thank you for your review @fccoelho!. I am going to merge this Pull Request. 🚀